caikit/runtime/utils/servicer_util.py

```diff
@@ -205,6 +208,17 @@ def validate_data_model(
             # and check to make that it is either a primitive protobufs type or that
             # we have a data model class that we can deserialize the protobufs with
             if not is_protobuf_primitive_field(field):
+
+                if field.message_type and field.message_type.GetOptions().map_entry:
```

Comment:
I guess the same question here as well - whether we want to remove `validate_data_model` altogether?

Reply:
Will be handled in a subsequent PR, not needed now